asapdiscovery-dataviz/asapdiscovery/dataviz/html_viz.py

```diff
@@ -57,9 +77,21 @@ def __init__(
         else:
             self.logger = logger
 
-        if target not in self.allowed_targets:
-            raise ValueError(f"Target must be one of: {self.allowed_targets}")
-        self.target = target
+        self.color_method = color_method
+        if self.color_method == "subpockets":
+            self.logger.info(f"Mapping interactive view by subpocket dict")
+        elif self.color_method == "bfactor":
```

Comment:
Call mutation fitness. I think will be more logical as b factor confused for crystallographic.